### ISSUE TITLE
mlterm doesn't generate mouse move events

### ISSUE THREAD

**Issue by dankamongmen** [dankamongmen/notcurses#2533]
Running `notcurses-input` in mlterm, we don't see mouse move events, only button presses. This is in mode 1006, but also seems to be the case for 1016 (see #1457).

**Comment by dankamongmen** [comment]
also, starting MLterm generates the input `\eP0+r\`, at least recently. well, this has been going on since at least 3.0.3.

**Comment by dankamongmen** [comment]
> also, starting MLterm generates the input `\eP0+r\`, at least recently. well, this has been going on since at least 3.0.3.

that's XTGETTCAP, right?

**Comment by klamonte** [comment]
Yes, that is a response to XTGETTCAP:
```
DCS 0 + r Pt ST for invalid requests.
          The strings are encoded in hexadecimal (2 digits per
          character).
```

I'm experiencing the same issue.  Seems like mlterm isn't handling any-event mode (1003), but is giving me button-event (1002).

**Comment by dankamongmen** [comment]
> Yes, that is a response to XTGETTCAP:
> 
> ```
> DCS 0 + r Pt ST for invalid requests.
>           The strings are encoded in hexadecimal (2 digits per
>           character).
> ```
> 
> I'm experiencing the same issue. Seems like mlterm isn't handling any-event mode (1003), but is giving me button-event (1002).

yep. i'm handling the (negative) XTGETTCAP now, but yeah, no any-event.

**Comment by klamonte** [comment]
That's a regression.  mlterm used to give me any-event.

**Comment by dankamongmen** [comment]
so do we flip a coin to see who writes up the issue on mlterm? =]

**Comment by klamonte** [comment]
Gotcha fam 👍

**Comment by dankamongmen** [comment]
> Gotcha fam +1

look at @klamonte speaking the patois! this made me break into a wide smile =]

**Comment by dankamongmen** [comment]
https://github.com/arakiken/mlterm/issues/34

**Comment by AutumnMeowMeow** [comment]
It's fixed in mlterm now.

(Hello from my new account going forward. :-) )
